Useful tools & utilities/Figma.md

```diff
@@ -0,0 +1,124 @@
+As a developer, understanding the fundamentals of the design tools used within your team is an essential skill. The notorious war between
+developers and designers is never-ending and design tools are the places where these two worlds collide. This raises the importance of using
+good design tools which provide a nice connection between the design and development process. Imagine if all the screen designs, components,
+different assets were stored on Google Drive and the review process and communication was handled through email/Slack. To get a single
+component, you'll have to go through several folders and then, you'll have to open your email/Slack to remind yourself about the discussions
+you had regarding the implementation. This isn't a big deal if you have to do it once in a day, however, this is a continuous and frequent
+process. Eventually, the project grows, new designs are requested, new folders and files are added and all of a sudden, handling designs
+like this sounds like a nightmare, right?
+
+![Designers vs Developers meme](/img/figma/figma_designers_vs_developers_meme.png "Designers vs Developers meme")
+
+[Figma](https://www.figma.com) is a cloud-based design tool which provides simple, effective and intuitive functionalities perfect for team
+collaboration. Similar to Google Docs, team members can easily communicate, collaborate and edit the same file in Figma in real time.
+
+## Benefits of using Figma
+
+- **Simple and intuitive**
+
+Most of us are already very familiar with using tools like Google Docs and Figma isn't much different from that. First-time Figma users can
+get quickly adjusted to using Figma and it won't take them too long to understand most of its functionalities.
+
+- **Easier team collaboration and communication**
+
+Communication and collaboration within Figma is extremely easy and convenient since Figma allows you to leave comments on the designs.
+Discussions can be opened in the comment's thread and if you have the right permissions for it, you could also @mention certain people in
+order to get a faster response from the discussion-relevant team members.
+
+- **Everything you need is at one place**
+
+Screen designs, components, assets, styles, comments, discussions and so much more can be found within a single Figma file. No need to
+include any third-party tools (unless the team feels like they would benefit from them), so basically, for all UI related stuff, you can
+refer to the Figma file. You want it, you open Figma, you got it, no further actions needed. :)
+
+- **Platform flexibility**
+
+If you have any operating system and you run any web browser, you are ready to use Figma. You can share, open, edit Figma files through your
```

Comment:
Maybe just rephrasing?
If your device has a web browser you are ready to use Figma.
Although I am not sure if this is true, good ol' IE (hopefully) can't run it 😅

Reply:
You are right, I wrote this part way too optimistically. 😅 I did some rephrasing and I provided more info to avoid incorrect assumptions about the browser requirements.